tenants/multi/templates/today-medical-design-development.marko

```diff
@@ -122,7 +122,13 @@ $ const socialMediaProviders = config.getAsArray('mdd.socialMediaLinks');
     />
 
     <!-- Footer/OptOut -->
-    <common-default-footer-block />
+    <common-default-footer-block
+      unsubscribe-cloud-page-id=854
+      company-name="Lynch Media"
+      please-contact-email="david@ien.com"
```

Comment:
Is this the correct contact-email? The other instance of this footer override I can find is using tom@cannabisequipmentnews.com

Reply:
@Shinsina david@ien.com is the correct email (I missed that change when I copied over the footer from cen newsletter, which is what they said it should be a copy of)

Reply:
@jwade1327 The text in the screenshot still says Tom Lynch over (what I presume) to be David Mantey.